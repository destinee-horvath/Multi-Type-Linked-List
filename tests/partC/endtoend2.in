NEW 3
01
10
00
NEW 4
bounce
bounce bounce bounce bounce
bounce bounce
bounce bounce bounce
INSERT 0 2 10
NEW 5
s
a
l
t
y
TYPE 2
a very invalid input
clearly invalid again
REMOVE 0
INSERT 1 2 {2}
VIEW-NESTED 1
TYPE 1
VIEW-NESTED 0 
NEW 3
discrete
is
hard
REMOVE 3
VIEW 3
DELETE 1 3
TYPE 1
