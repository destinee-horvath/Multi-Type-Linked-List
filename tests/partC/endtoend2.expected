List 0: 1 -> 10 -> 0
List 1: bounce -> bounce bounce bounce bounce -> bounce bounce -> bounce bounce bounce
List 0: 1 -> 10 -> 10 -> 0
List 2: s -> a -> l -> t -> y
char -> char -> char -> char -> char
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
List 0 has been removed.

Number of lists: 2
List 1
List 2
Nested 1: bounce -> bounce bounce bounce bounce -> {List 2} -> bounce bounce -> bounce bounce bounce
bounce -> bounce bounce bounce bounce -> {s -> a -> l -> t -> y} -> bounce bounce -> bounce bounce bounce
string -> string -> reference -> string -> string
INVALID COMMAND: VIEW-NESTED
List 3: discrete -> is -> hard
List 3 has been removed.

Number of lists: 2
Nested 1
List 2
INVALID COMMAND: VIEW
Nested 1: bounce -> bounce bounce bounce bounce -> {List 2} -> bounce bounce bounce
string -> string -> reference -> string
